packages/angular/projects/cloudinary-library/src/lib/cloudinary-video.component.ts

```diff
@@ -83,6 +83,12 @@ export class CloudinaryVideoComponent implements OnInit, OnChanges, OnDestroy {
       this.plugins,
       this.getVideoAttributes()
       );
+
+    // check if video should be muted. We need to take care of this here since Angular has a bug with binding the muted
+    // attribute
+    if (this.muted) {
+      this.el.nativeElement.children[0].muted = true;
```

Comment:
Do we expect any issues with SSR with this approach? 
Does the element exist?

Reply:
not a concern on angular's end. Universal is smart enough to deal with this.